index.html

```diff
@@ -107,77 +109,82 @@ <h4 class="works__single__title">Multi-Post Stories</h4>
           alt="project 1"
         />
 
-        <h4 class="works__single__title">Tonic</h4>
+        <div class="works__single__holder">
+          <h4 class="works__single__title">Tonic</h4>
 
-        <div class="works__single__details">
-          <p
-            class="works__single__details__single works__single__details__single--dark"
-          >
-            CANPOY
-          </p>
-          <div class="dot"></div>
-          <p
-            class="works__single__details__single works__single__details__single--light"
-          >
-            Back End Dev
-          </p>
-          <div class="dot"></div>
-          <p
-            class="works__single__details__single works__single__details__single--light"
-          >
-            2015
+          <div class="works__single__details">
+            <p
+              class="works__single__details__single works__single__details__single--dark"
+            >
+              CANPOY
+            </p>
+            <div class="dot"></div>
+            <p
+              class="works__single__details__single works__single__details__single--light"
+            >
+              Back End Dev
+            </p>
+            <div class="dot"></div>
+            <p
+              class="works__single__details__single works__single__details__single--light"
+            >
+              2015
+            </p>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
           </p>
-        </div>
```

Comment:
- [ ] Consider using the correct margins for these elements to space them out properly as they are too close to one another on the mobile version. I suggest you check the Figma template to get the correct margin and apply them to the `CSS`

Your design:
<img width="189" alt="Loom_UyPfxhochZ" src="https://user-images.githubusercontent.com/81468675/149225357-c7ea81b2-1be0-43d7-9c41-f29f520db579.png">

Figma design:
<img width="146" alt="Loom_QhxA02T8Gd" src="https://user-images.githubusercontent.com/81468675/149225450-a608e19e-5b53-4948-b09f-14053861366c.png">

- [OPTIONAL] I'm adding this as optional because I can see from the mobile version that you have only two project cards and that is why there are only two cards in the desktop version instead of four as in the Figma template. Please consider adding the remaining two protect cards as is seen in the Figma template.